lib/spack/spack/package_base.py

```diff
@@ -926,6 +922,24 @@ def global_license_file(self):
             self.global_license_dir, self.name, os.path.basename(self.license_files[0])
         )
 
+    @property
+    def keep_werror(self) -> Optional[str]:
+        """Keep -Werror flags, matches config:flags:keep_werror to override config
+
+        Valid return values are 'all', 'specific', 'none', and ``None``. ``None`` means
+        we'll respect the user's configuration.
+
+        NOTE: return type should be Optional[Literal['all', 'specific', 'none']] in Python
+        3.8+, but we still support 3.6.
```

Comment:
This prob doesn't need to make it into the docstring.

Reply:
fixed